### ISSUE TITLE
[Question] Does Ergo communicate with other IRC servers? 

### ISSUE THREAD

**Issue by paulo-roger** [ergochat/ergo#2186]
Sorry for the newbie question.

Ergo doesn't have a Server to Server dedicated port (eg. 7000) So  I was wondering if after setting up ergo I would be able to talk to people in other servers, like libera for instance.

**Comment by slingamn** [comment]
This is not supported. See #1532 for discussion; note that you would not be allowed to federate with Libera even if you were running the same IRC server software as them :-)

Thanks for your interest in the project!

**Comment by paulo-roger** [comment]
> This is not supported. See #1532 for discussion; note that you would not be allowed to federate with Libera even if you were running the same IRC server software as them :-)
> 
> Thanks for your interest in the project!

ohh I was thinking IRC was more like xmpp or activitypub with fre federation. So thats not the case?
Usually people setup servers to create their own rooms for people they already know?

Thanks for being so kind!

**Comment by slingamn** [comment]
Yeah, that's right. IRC originally had untrusted federation but it turned out that the protocol wasn't really secure against malicious nodes, so operators began restricting federation to trusted nodes. Here are some notes on that history: https://daniel.haxx.se/irchistory.html

Depending on your use case, you could set up your own Ergo, or create your own channel on [irc.ergo.chat](https://irc.ergo.chat/about-network).

**Comment by paulo-roger** [comment]
Thank you very much!
